tasks/networkmanager.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: Ensure the `VXLAN` configuration is present within `/etc/sysconfig/network-scripts/`
+  ansible.builtin.template:
+    src: "templates/nmconnection-vxlan.j2"
+    dest: "/etc/NetworkManager/system-connections/{{ item.bridge if item.bridge is defined else item.phys_dev | default(vxlan_phys_dev)}}-vxlan{{ item.vni | default(vxlan_vni) }}.nmconnection"
```

Comment:
Any reason not to use item.device as the device name? I think normally it would just be vxlanNNN

Reply:
I was thinking about avoiding confusion between the name in NM and the device name. It is possible to have them be the same. Here, the top shows the current naming, and the bottom shows your suggestion. I'm happy to go with either:
![image](https://user-images.githubusercontent.com/112560678/226871400-01631e36-8576-4ac6-b382-046198961477.png)

Reply:
I guess I don't see a reason to make them different, but maybe I'm missing something. It's probably not worth changing though.

Reply:
Had to change it to test it so it's in now